### DISCUSSION TITLE
asking about comfyui

### DISCUSSION THREAD

**Discussion by sumail3399** [Comfy-Org/ComfyUI#4441] [General]
i just installed comfyui and a workflow, and ran into a problem in the section

Prompt outputs failed validation
KSampler:
- Return type mismatch between linked nodes: scheduler, ['normal', 'karras', 'exponential', 'sgm_uniform', 'simple', 'ddim_uniform', 'beta', 'AYS SDXL', 'AYS SD1', ' AYS SVD', 'GITS[coeff=1.2]'] != ['normal', 'karras', 'exponential', 'sgm_uniform', 'simple', 'ddim_uniform', 'beta']
the workflow creator told me to Rollback [ComfyUI Image Saver] to version v1.1.0

does anyone know how to do it?, thanks

**Comment by ltdrdata** [comment]
What is your workflow?

**Reply by sumail3399** [reply]
problem resolved, the creator re-uploaded the json file, and it worked smoothly 🐱‍🏍

**Comment by Moorer624** [comment]
How was this resolved? I have the same exact problem with my own workflows when I added the KSampler Config node from rgthree.
